go.mod

```diff
@@ -1,22 +1,22 @@
 module github.com/coredhcp/coredhcp
 
-go 1.12
+go 1.13
```

Comment:
we can probably bump directly to 1.14

Reply:
Is there something interesting we want from 1.14 ? I bumped for `%w` and error wrapping, I don't know if there are other goodies in 1.14 that we want to use